l10n_it_account/models/account_group.py

```diff
@@ -73,7 +73,7 @@ def get_account_balance_sign(self):
         return 1
 
     def get_group_accounts(self):
-        """ Retrieves every account from `self` and `self`'s subgroups. """
```

Comment:
puoi mettere le modifica di questo modulo (`l10n_it_account`), se necessarie, in un commit separato?

Reply:
la modifica era stata aggiunta nel commit `black, isort, prettier`, eliminandola pre-commit non fallisce.